test/core/tsi/alts/zero_copy_frame_protector/alts_zero_copy_grpc_protector_test.cc

```diff
@@ -194,6 +197,13 @@ static void seal_unseal_small_buffer(tsi_zero_copy_grpc_protector* sender,
         1;
     grpc_slice_buffer_move_first(&var->protected_sb, staging_sb_size,
                                  &var->staging_sb);
+    // TODO(gtcooke94) call the test with and without this
+    if (do_frame_size_read) {
```

Comment:
Do we need the do_frame_size_read boolean or the TODO now that the function is stateless? Can we always just call the read frame size API?

Reply:
In the small buffer case I think we can always call it